src/gui/mzroll/mzroll.pro

```diff
@@ -238,6 +238,7 @@ HEADERS +=  stable.h \
 SOURCES += mainwindow.cpp  \
 database.cpp \
  plotdock.cpp \
+ spectralhit.cpp \
```

Comment:
how was this not present in the pro file already :o

Reply:
Because I created it with that commit. The declaration and the definitions were both within `spectralhit.h` header file.